templates/locales-banner.njk

```diff
@@ -14,11 +14,15 @@ It receives:
             <ul class="govuk-language-select__list">
             {% if languageEnabled %}
                {% for l in languages %}
-                <li class="govuk-language-select__list-item">
-                    <a href="{{lang2url.addLangToUrl(currentUrl, l.IsoCode, true)}}" lang={{l.IsoCode}} rel="alternate" class="govuk-link" >
-                        <span>{{l.Name}}</span>
-                    </a>
-                </li>
+                 <li class="govuk-language-select__list-item">
+                 {% if lang == l.IsoCode %}
```

Comment:
would it be worth testing that `lang` is actually available ?
Ex
```suggestion
                 {% if lang and lang == l.IsoCode %}
```
this way even when lang is not pushed to the template, it falls cleanly back to the usual case

Reply:
done